### PR TITLE
For #18629: ETP SmartBlock

### PR DESCRIPTION
**Pull request by eliserichards**
For #18629

https://user-images.githubusercontent.com/773158/128058475-ad09ce5c-c1ab-444a-8eea-4fec559ec716.mp4

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by Amejia481** [comment]
🎉

File: app/src/main/java/org/mozilla/fenix/trackingprotection/TrackerBuckets.kt
Comment (eliserichards) [inline app/src/main/java/org/mozilla/fenix/trackingprotection/TrackerBuckets.kt:99]: For now we're just pretending that the category works

File: app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionPanelView.kt
Comment (eliserichards) [inline app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionPanelView.kt:284]: Todo for ln 279

File: app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionPanelView.kt
Comment (Amejia481) [inline app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionPanelView.kt:152]: I think we can have content that was unblock in any category  [see](https://github.com/mozilla-mobile/android-components/pull/10676/files#diff-332e50b45d5cf7dda87dd76b059e54ac5c9050b9c7af6cac021ef143249c5f7dR22)

**Comment by Amejia481** [comment]
Let's wait until we can land https://github.com/mozilla-mobile/fenix/issues/19886 to merge this.

**Comment by eliserichards** [comment]
Failing test: 
```
    InactiveTabsControllerTest. WHEN expanded THEN track telemetry event
```

**Comment by Amejia481** [comment]
> Failing test:
> 
> ```
>     InactiveTabsControllerTest. WHEN expanded THEN track telemetry event
> ```

I think this is a flaky tests

**Review by Amejia481 — APPROVED** [review]

**Pull request merged**
